### ISSUE TITLE
Getting error while building

### ISSUE THREAD

**Issue by paly2** [asciidoctor/jekyll-asciidoc#44]
Hello,

I get the error `jekyll 3.1.2 | Error:  undefined method getConverterImpl' for #<Jekyll::Site:0xa1cc44c>` when I build with the `jekyll-asciidoc` gem in `_config.yml`...

Here's my `_config.yml` :
```
source: .
destination: ./_site
timezone: "Europe/Paris"
encoding: "utf-8"
show_drafts: false
future: true
detach: false
gems: [jekyll-asciidoc]
```

Here's the output of `gem list --local` :
```
*** LOCAL GEMS ***

asciidoctor (1.5.4)
bigdecimal (1.2.0)
colorator (0.1)
ffi (1.9.10)
io-console (0.4.2)
jekyll (3.1.2)
jekyll-asciidoc (1.0.0)
jekyll-sass-converter (1.4.0)
jekyll-watch (1.3.1)
json (1.7.7)
kramdown (1.9.0)
liquid (3.0.6)
listen (3.0.6)
mercenary (0.3.5)
minitest (4.3.2)
psych (2.0.0)
rake (0.9.6)
rb-fsevent (0.9.7)
rb-inotify (0.9.7)
rdoc (4.0.0)
rouge (1.10.1)
safe_yaml (1.0.4)
sass (3.4.21)
test-unit (2.0.0.0)
```

Could you help me ?
Thanks,
paly2

**Comment by mojavelinux** [comment]
This will be fixed by #36.

**Comment by mojavelinux** [comment]
1.0.1 is out.
